Source/Scene/GroundPolylinePrimitive.js

```diff
@@ -0,0 +1,759 @@
+     * @param {Boolean} [options.show=true] Determines if this primitive will be shown.
+     * @param {Boolean} [options.vertexCacheOptimize=false] When <code>true</code>, geometry vertices are optimized for the pre and post-vertex-shader caches.
+     * @param {Boolean} [options.interleave=false] When <code>true</code>, geometry vertex attributes are interleaved, which can slightly improve rendering performance but increases load time.
+     * @param {Boolean} [options.compressVertices=true] When <code>true</code>, the geometry vertices are compressed, which will save memory.
+     * @param {Boolean} [options.releaseGeometryInstances=true] When <code>true</code>, the primitive does not keep a reference to the input <code>geometryInstances</code> to save memory.
+     * @param {Boolean} [options.allowPicking=true] When <code>true</code>, each geometry instance will only be pickable with {@link Scene#pick}.  When <code>false</code>, GPU memory is saved.
+     * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready. If false initializeTerrainHeights() must be called first.
+     * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Determines if this primitive's commands' bounding spheres are shown.
+     * @param {Boolean} [options.debugShowShadowVolume=false] For debugging only. Determines if the shadow volume for each geometry in the primitive is drawn. Must be <code>true</code> on
+     *                  creation for the volumes to be created before the geometry is released or options.releaseGeometryInstance must be <code>false</code>.
+     *
+     */
+    function GroundPolylinePrimitive(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        /**
+         * The geometry instances rendered with this primitive. This may
+         * be <code>undefined</code> if <code>options.releaseGeometryInstances</code>
+         * is <code>true</code> when the primitive is constructed.
+         * <p>
+         * Changing this property after the primitive is rendered has no effect.
+         * </p>
+         *
+         * @readonly
+         * @type {Array|GeometryInstance}
+         *
+         * @default undefined
+         */
+        this.geometryInstances = options.geometryInstances;
+        this._hasPerInstanceColors = true;
+
+        var appearance = options.appearance;
+        if (!defined(appearance)) {
+            appearance = new PolylineMaterialAppearance();
+        }
+        /**
+         * The {@link Appearance} used to shade this primitive. Each geometry
+         * instance is shaded with the same appearance.  Some appearances, like
+         * {@link PolylineColorAppearance} allow giving each instance unique
+         * properties.
+         *
+         * @type Appearance
```

Comment:
Here and above, should this be `PolylineColorAppearance`?

Reply:
It can also be `PolylineMaterialAppearance`.